verilog/formatting/formatter_test.cc

```diff
@@ -1198,6 +1198,20 @@ static const std::initializer_list<FormatterTestCase> kFormatterTestCases = {
      "    end\n"
      "  join_any\n"
      "endtask\n"},
+    {"a=b&~(c<<d);",
+     "a = b & ~(c << d);\n"},
+    {"a=~~~~b;",
+     "a = ~~~~b;\n"},
+    {"a = ~ ~ ~ ~ b;",
+     "a = ~~~~b;\n"},
+    {"a   =   ~--b;",
+     "a = ~--b;\n"},
+    {"a   =   ~ --b;",
+     "a = ~--b;\n"},
+    {"a = ~ ++ b;",
+     "a = ~++b;\n"},
+    {"a=--b- --c;",
+     "a = --b - --c;\n"}
```

Comment:
Wasn't sure if these test would pass as is, but if it complains about parse failure because the context is unknown, I recommend making these look like: `parameter int a = ...;`.  See similar test cases near the comment `// parameter test cases`.  Please group these tests there.

Reply:
These tests did pass, but it still makes sense to group them with the parameter tests, so I did just that